tests/framework/ensembleModelTests/tests

```diff
@@ -91,5 +91,10 @@
    UnorderedCsv = 'IndexInputOutput/step_out.csv IndexInputOutput/step_out_2.csv IndexInputOutput/step_out_4.csv IndexInputOutput/step_out_6.csv IndexInputOutput/step_out_8.csv'
  [../]
 
+ [./testEMConnection]
+   type = 'RavenFramework'
+   input = 'test_em_connection.xml'
+   output = 'checkConnection/1-plotData_scatter.png'
```

Comment:
either you check the plot with an Image Checker or you dump a csv for comparing with a gold file.

Second option is the preferred  one (in case, remove the plot that it will be useless)

Reply:
fixed.